unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Tests/BackpackEditorHUDControllerShould.cs

```diff
@@ -182,6 +186,28 @@ public void EquipBodyShapeCorrectly()
             backpackEmotesSectionController.Received(1).SetEquippedBodyShape(testUserProfileModel.avatar.bodyShape);
         }
 
+        [Test]
+        [TestCase("eyes")]
+        [TestCase("hair")]
+        [TestCase("bodyshape")]
+        [TestCase("non_existing_category")]
+        public void ToggleSlotCorrectly(string slotCategory)
+        {
+            // Act
+            avatarSlotsView.OnToggleAvatarSlot += Raise.Event<Action<string, bool, bool>>(slotCategory, true, true);
+
+            // Assert
+            view.Received(1).SetColorPickerVisibility(true);
+
+            if (slotCategory == "non_existing_category")
+            {
+                view.DidNotReceive().SetColorPickerValue(Arg.Any<Color>());
+                return;
+            }
+
+            view.Received(1).SetColorPickerValue(Arg.Any<Color>());
```

Comment:
you should be asserting specific values for the given cases

Reply:
Done! https://github.com/decentraland/unity-renderer/pull/5028/commits/2db787356a6b9b5008424c4d66468ef601a6aa9f